state/state.go

```diff
@@ -54,6 +56,15 @@ type HelmState struct {
 	runner helmexec.Runner
 }
 
+// SubHelmfileSpec defines the subhelmfile path and options
+type SubHelmfileSpec struct {
+	Path      string   //path or glob pattern for the sub helmfiles
+	Selectors []string //chosen selectors for the sub helmfiles
+	Inherits  bool     //do the sub helmfiles inherits from parent selectors
+}
+
+const InheritsYamlValue = "inherits"
```

Comment:
Nit: Can be `SelectorsInherits` so that it looks a bit more consistent with golang community's convention that uses `Where + What` for const names

Reply:
This can be removed completely if we go this way https://github.com/roboll/helmfile/pull/567/files#r281003831